app/Http/Controllers/RoutesController.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Routes;
+use App\Coords;
+use Illuminate\Http\Request;
+use App\Http\Resources\RouteCollection;
+
+use function GuzzleHttp\json_encode;
+
+class RoutesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        
+        return routes::find(2);
+        
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        
+        
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Routes  $routes
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Routes $routes, Request $request)
+    {
+        $coordData = coords::all();
+        
+        $returnArray = [];
+        $point = null;
+
+        foreach ($coordData as $item){
+            $routeData = routes::where('wall_location', $item->id)->get();
+            $routeDiff = $routeData[0]->difficulty;
+
+            if($routeDiff == "5.7"){
```

Comment:
A `switch` statement might be a little cleaner here. It's functionally the same, but conventionally preferred for this many conditionals. Again, just a matter of preference.